cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/ArgumentHolder.kt

```diff
@@ -68,4 +68,6 @@ interface ArgumentHolder : Holder<Expression> {
     operator fun minusAssign(node: Expression) {
         removeArgument(node)
     }
+
+    fun hasArgument(expression: Expression): Boolean
```

Comment:
@konradweiss this is needed in order to check the context of a call expression in order to derive the function type. But adding this "polutes" the PR somewhat because I need to change all files that implement `ArgumentHolder`. I guess it would make sense to have this in a separate PR?